corehq/apps/es/users.py

```diff
@@ -64,13 +64,17 @@ def show_only_inactive(self):
         return query.is_active(False)
 
 
-def domain(domain, allow_enterprise=False):
+def domain(domain, allow_enterprise=False, allow_enterprise_controlled_domains=False):
+    from corehq.apps.enterprise.models import EnterprisePermissions
     domain_list = [domain]
     if allow_enterprise:
-        from corehq.apps.enterprise.models import EnterprisePermissions
         config = EnterprisePermissions.get_by_domain(domain)
         if config.is_enabled and domain in config.domains:
             domain_list.append(config.source_domain)
+    if allow_enterprise_controlled_domains:
```

Comment:
Is there a reason you need this conditional since you check `config.is_enabled` here anyway?

Reply:
hm maybe I'm not following, but that check is only if the param is explicitly set to true to enable that functionality of adding the mirrored domains